### PR TITLE
Use Set, to ensure not repeat items

### PR DESCRIPTION
**Pull request by FCesar**

### CODE REVIEW

File: bin/runnerman.js [hunk 1]
```diff
@@ -6,18 +6,22 @@ const { runnerman } = require("..");
...
-  .option('-s, --suite <type>')
+  .option('-s, --suite <value>', '', collect, [])
   .option('-i, --iterations <type>')
   .parse(process.argv);
 
 (async function main(program) {
   if (program.collection === '' ||
       program.environment === '' ||
-      program.suite === '')
+      program.suite === undefined || program.suite.length === 0)
```
Comment (brunofurmon) [inline bin/runnerman.js:24]:
```suggestion
      program.suite === undefined ||
      program.suite.length === 0)
```

File: bin/runnerman.js [hunk 2]
```diff
@@ -42,7 +51,7 @@ program
 
   let exitCode = 0;
 
-  if (summaries.some(x => x.run.failures.length > 0))
+  if (summaries.some(y => y.run.failures.length > 0))
     exitCode = 1
```
Comment (brunofurmon) [inline bin/runnerman.js:55]:
```suggestion
  if (summaries.some(y => y.run.failures.length > 0)) {
       exitCode = 1;
  }
  
```

File: bin/runnerman.js [hunk 3]
```diff
@@ -28,7 +33,12 @@ program
 
   const iterations = parseInt(program.iterations) || 1;
 
-  const suites = getFilesInFolderPerExtension(program.suite, "suite");
+  const suites = new Set();
+
+  for(const item of program.suite) {
```
Comment (brunofurmon) [inline bin/runnerman.js:38]:
```suggestion
  for (const item of program.suite) {
```

File: bin/runnerman.js [hunk 4]
```diff
@@ -28,7 +33,12 @@ program
...
+    var items = await getFilesInFolderPerExtension(item, "suite");
```
Comment (brunofurmon) [inline bin/runnerman.js:39]:
```suggestion
    const items = await getFilesInFolderPerExtension(item, "suite");
```

Comment (FCesar) [reply bin/runnerman.js:37]: Resolved this point in https://github.com/FCesar/runnerman/pull/4

File: bin/runnerman.js (hunk 3 above)
Comment (FCesar) [reply bin/runnerman.js]: Resolved this point in https://github.com/FCesar/runnerman/pull/4

**Pull request merged**